### PR TITLE
Add OpnForm

### PR DESCRIPTION
**Pull request by JhumanJ**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/opnform.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description: "Beautiful open-source form builder"
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
+  - Nodejs
+  - Docker
+tags:
+  - Polls and Events
+depends_3rdparty: false
```
Comment (nodiscc) [inline software/opnform.yml:14]:
```suggestion
```

This is already the default value, can be removed

File: software/opnform.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+demo_url: "https://opnform.com/forms/create/guest"
+stargazers_count: 2014
+updated_at: '2024-10-14'
+archived: false
```
Comment (nodiscc) [inline software/opnform.yml:18]:
```suggestion
```

These fields are kept up-to-date automatically, no need to include them.

File: software/opnform.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
+name: "OpnForm"
+website_url: "https://opnform.com"
+source_code_url: "https://github.com/JhumanJ/opnform"
+description: "Beautiful open-source form builder"
```
Comment (nodiscc) [inline software/opnform.yml:5]:
```suggestion
description: "Beautiful open-source form builder."
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by JhumanJ** [comment]
@nodiscc thanks for the fast review - all applied.

File: software/opnform.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (Ki-er) [inline software/opnform.yml:1]:
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Very very pedantic review, lets remove the empty whitespace.

**Comment by JhumanJ** [comment]
> Very very pedantic review, lets remove the empty whitespace.

Done!

**Pull request merged**